packages/wpcom-template-parts/src/types.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module '@automattic/components' {
+	type Props = {
+		className?: string;
+		size?: { width: number; height: number };
+	};
+
+	export const WordPressWordmark: React.FunctionComponent< Props >;
+}
```

Comment:
Is this something we can declare in the `@automattic/components` package itself?